mos_session/jwebbinar7_nirspecmos.ipynb

```diff
@@ -18,7 +18,8 @@
     "\n",
```

Comment:
<p>Hi @jmuzerolle, before I do the review in full, can you add code for downloading the cached NIRSpec files used in this notebook from Box? The paths in this cell are not accessible to others. (Your JWebbinar 5 notebook has the same issue, but let's just do this notebook for now.)</p><p> </p><p>If you don't remember how, you can use a similar strategy as you did in the "Download Data" section of your JWebbinar 5 notebook. Just substitute in a new Box link and set the symlink to <code>output_dir</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>